tests/test_cells.py

```diff
@@ -4,6 +4,134 @@
 from vtk.util.numpy_support import vtk_to_numpy
 
 import pyvista
+from pyvista import Cell, CellType
+from pyvista.examples import (
+    load_airplane,
+    load_explicit_structured,
+    load_hexbeam,
+    load_rectilinear,
+    load_structured,
+    load_tetbeam,
+    load_uniform,
+)
+
+
+class Test_Cell:
+
+    grids = [
+        load_hexbeam(),
+        load_airplane(),
+        load_rectilinear(),
+        load_structured(),
+        load_tetbeam(),
+        load_uniform(),
+    ]
+    if pyvista._vtk.VTK9:
+        grids.append(load_explicit_structured())
+
+    ids = list(map(type, grids))
+
+    def test_bad_init(self):
+        with pytest.raises(TypeError, match="must be of vtkCell"):
+            _ = Cell(1)
+
+    @pytest.mark.parametrize("grid", grids, ids=ids)
+    def test_cell_attribute(self, grid):
+        assert isinstance(grid.cell, list)
+        assert len(grid.cell) == grid.n_cells
+        assert all(t == Cell for t in map(type, grid.cell[:]))
+
+    @pytest.mark.parametrize("grid", grids, ids=ids)
+    def test_cell_type_is_inside_enum(self, grid):
+        assert grid.cell[0].type in CellType
+
+    types = [
+        CellType.HEXAHEDRON,
```

Comment:
nitpick suggestion: I think it would be better to collocate these variables with `grids` since I had to scroll up and down to compare.

Reply:
Yes, I changed it